test/tick-processor/test-tick-processor-unknown.js

```diff
@@ -21,7 +21,7 @@ const base = require('./tick-processor-base.js');
 // Unknown checked for to prevent flakiness, if pattern is not found,
 // then a large number of unknown ticks should be present
 base.runTest({
-  pattern: /LazyCompile.*\[eval\]:1|.*%  UNKNOWN/,
+  pattern: /LazyCompile.*\[eval]:1|.*%  UNKNOWN/,
```

Comment:
I personally find the explicit escaping more readable but I don't feel too strongly about it.